samples/bookinfo/kube/route-rule-ratings-mysql.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: config.istio.io/v1alpha2
+kind: RouteRule
+metadata:
+  name: ratings-test-v2-mysql
+spec:
+  destination:
+    name: ratings
+  precedence: 3
+  route:
+  - labels:
+      version: v2-mysql
```

Comment:
same as route-rule-ratings-db.yaml with a -mysql added to select the mysql one and they can coexist